src/app/demo/typescript/typescript.component.ts

```diff
@@ -1,6 +1,6 @@
-import {Component} from '@angular/core';
-import {typeScriptWithConsoleLog, withDeps} from '../../exercise/helpers/helpers';
-import {ng2tsConfig} from '../../../../ng2ts/ng2ts';
+import { Component, HostBinding } from '@angular/core';
```

Comment:
@kirjs if this is the case, we need to change tslint.json to remove "check-module" from "whitespace"